app/javascript/css/pages/track-dashboard.css

```diff
@@ -0,0 +1,158 @@
+@import "../styles";
+
+#page-track-dashboard {
+    & .track-nav {
+        --nav-height: 40px;
+
+        @apply bg-white py-12;
+        & .container {
+            @apply flex;
+        }
+
+        & .back {
+            @apply block mr-24;
+            line-height: var(--nav-height);
+        }
+
+        & .c-track-icon {
+            height: var(--nav-height);
+            width: calc(var(--c-track-icon-width-ratio) * var(--nav-height));
+            @apply mr-20;
```

Comment:
We're gonna have to update this to not use a `margin-right` to set the tabs placing, and have the tabs start in a consistent position on all tracks.

Reply:
Out of interest, why? The longest track title is long compared to the shortest, so they'll probably feel a bit like they're floating?

Reply:
Just so the nav can stay in a consistent place.

I've tested it a little bit and the nav does look a bit funky floating with shorter language names.

Let's stick with this for now.